src/components/vaults/community/vaultDetails/profileVaultCard/ProfileVaultCard.tsx

```diff
@@ -84,24 +78,29 @@ export default function ProfileVaultCard(props: Props) {
         <div className='space-y-4'>
           <InfoRow label='APR'>
             <FormattedNumber
-              amount={metrics.apr}
+              amount={Number(details.metrics.apr)}
               options={{ minDecimals: 2, maxDecimals: 2, suffix: '%' }}
               className='text-sm'
             />
           </InfoRow>
           <InfoRow label='TVL'>
             <DisplayCurrency
-              coin={BNCoin.fromDenomAndBigNumber('usd', BN(metrics.tvl))}
+              coin={BNCoin.fromDenomAndBigNumber('usd', BN(details.metrics.tvl))}
               className='text-sm'
             />
           </InfoRow>
           {isOwner ? (
             <>
               <InfoRow label='Accrued PnL'>
                 <DisplayCurrency
-                  coin={BNCoin.fromDenomAndBigNumber('usd', BN(0))}
-                  // TODO:conditional classnames for profit/loss
-                  className={classNames('text-profit text-sm')}
+                  coin={BNCoin.fromDenomAndBigNumber(
+                    'usd',
+                    BN(details.performance_fee_state.accumulated_pnl),
+                  )}
+                  className={classNames('text-sm text-white', {
+                    'text-profit': BN(details.performance_fee_state.accumulated_pnl).isPositive(),
+                    'text-loss': BN(details.performance_fee_state.accumulated_pnl).isNegative(),
+                  })}
```

Comment:
Since 0 is also considered positive:
```suggestion
                  className={classNames('text-sm text-white', 
                     BN(details.performance_fee_state.accumulated_pnl).isGreaterThan(0) && 'text-profit',
                     BN(details.performance_fee_state.accumulated_pnl).isLessThan(0) && 'text-loss',
                  })}
```